### DISCUSSION TITLE
PolyData.cell returns an error

### DISCUSSION THREAD

**Discussion by yannis-df** [pyvista/pyvista#4021] [Q&A] [answered]
Hello,

When calling "cell" from PolyData I get the following error:

```
sphere = pv.Sphere()
sphere.cell[0]
>>>AttributeError: 'PolyData' object has no attribute 'cell'
```
[The documentation showing that it's supposed to work](https://docs.pyvista.org/api/core/_autosummary/pyvista.Cell.points.html)

My python version is 3.9.12
My pyvista version is 0.35.1

More specifically, what i'm trying to do is to remove smaller bits of a mesh that are not connected to the main part.
`mesh = pv.read(filename="my_file")`

![github_mesh](https://user-images.githubusercontent.com/87186644/218799885-0f7f5284-7049-4ac4-a060-9cfdd902732f.PNG)

For that, I am using the following function:
`mesh_extract = mesh.connectivity(largest=True)`

The issue is that it does not remove the points from the PolyData object (though the cells are gone). These points are hidden in the viewer but still exist:

![github_mesh2](https://user-images.githubusercontent.com/87186644/218800188-32a67d0a-5f6e-4b0d-86b2-5b62129214be.PNG)

In the image above, the colored points are points taken from the PolyData, and highlighted using pyvista viewer (example code below):

```
arbitrary_max = 100
arbitrary_min = 10
my_bool = (mesh_extract.points[:, 0]<arbitrary_max)&((mesh_extract.points[:, 0]>arbitrary_min)
p = pv.Plotter()
p.add_mesh(mesh_extract, color="white")
p.add_points(mesh_extract.points[my_bool], color="purple")
p.show_axes()
p.show()
```

In order to remove these points, I thought about removing any point that is not connected to a cell. This means that i need to know if a specific point is connected to any cell, but I can't find any info about this particular task. I might be able to extract this info if i had access to cell data (which points are associated with a specific cell), but it doesn't seem to work as explained above.

Thank you in advance for any help provided.

**Comment by banesullivan** [comment, marked as answer]
You'll need to update to PyVista v0.38. you're on `0.35.1` which is a bit dated and does not have the new `cell` attribute

**Reply by MatthewFlamm** [reply]
And don't do this:

`sphere.cell[0]`

Do this:

`sphere.get_cell(0)`

See #3971.

**Reply by yannis-df** [reply]
Thank you both for the answers. If you happen to also know how to find cells that are linked to a specific point, please let me know.
It's faster to run through points rather than cells for what i'm trying to do.
